platform/server/elasticsearch/AdminClient.ts

```diff
@@ -1,10 +1,17 @@
 import { Client } from 'elasticsearch';
+import { callAPI } from './call_api';
+
+type CallAPIOptions = { wrap401Errors?: boolean };
+type CallAPIClientParams = { [key: string]: any };
 
 export class AdminClient {
   constructor(private readonly client: Client) {}
 
-  call(endpoint: string, clientParams = {}, options = {}): any {
-    this.client.search({});
-    return {};
+  call(
+    endpoint: string,
+    clientParams: CallAPIClientParams = {},
```

Comment:
__optional nit:__ since this part of API started to take a final shape it would be great if we can add some JSDoc explaining what this method does and what the difference between `clientParams` and `options` is. If not in this PR then maybe in the upcoming ones.

Reply:
This is a good suggestion. I can add this.